Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
   static const char TR_PER_PIXEL_LIGHTING_DESCRIPTION[] = QT_TR_NOOP(
       "Calculates lighting of 3D objects per-pixel rather than per-vertex, smoothing out the "
-      "appearance of lit polygons and making individual triangles less noticeable.\nRarely causes "
-      "slowdowns or graphical issues.\n\nIf unsure, leave this unchecked.");
+      "appearance of lit polygons and making individual triangles less noticeable.\n\nRarely "
+      "causes slowdowns or graphical issues.\n\nIf unsure, leave this unchecked.");
   static const char TR_WIDESCREEN_HACK_DESCRIPTION[] = QT_TR_NOOP(
-      "Forces the game to output graphics for any aspect ratio.\nUse with \"Aspect Ratio\" set to "
-      "\"Force 16:9\" to force 4:3-only games to run at 16:9.\nRarely produces good results and "
-      "often partially breaks graphics and game UIs.\nUnnecessary (and detrimental) if using any "
+      "Forces the game to output graphics at any given aspect ratio.\n\nUse with \"Aspect Ratio\" "
+      "set to \"Force 16:9\" to force 4:3-only games to run at 16:9. Rarely produces good results "
+      "and often partially breaks graphics and game UIs. Unnecessary (and detrimental) if using any "
       "AR/Gecko-code widescreen patches.\n\nIf unsure, leave this unchecked.");
   static const char TR_REMOVE_FOG_DESCRIPTION[] =
-      QT_TR_NOOP("Makes distant objects more visible by removing fog, thus increasing the overall "
-                 "detail.\nDisabling fog will break some games which rely on proper fog "
-                 "emulation.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Makes distant objects more visible by removing fog, thus increasing overall "
+                 "detail.\n\nDisabling fog will break some games which rely on proper fog "
+                 "emulation.\n\nIf unsure, leave unchecked.");
   static const char TR_3D_MODE_DESCRIPTION[] = QT_TR_NOOP(
-      "Selects the stereoscopic 3D mode. Stereoscopy allows you to get a better feeling "
-      "of depth if you have the necessary hardware.\nSide-by-Side and Top-and-Bottom are "
+      "Selects the stereoscopic 3D mode. Stereoscopy allows you to get a better perception "
+      "of depth if you have the necessary hardware. This option heavily decreases emulation "
+      "speed and sometimes causes issues.\n\nSide-by-Side and Top-and-Bottom are "
       "used by most 3D TVs.\nAnaglyph is used for Red-Cyan colored glasses.\nHDMI 3D is "
-      "used when your monitor supports 3D display resolutions.\nHeavily decreases "
-      "emulation speed and sometimes causes issues.\n\nIf unsure, select Off.");
+      "used when your monitor supports 3D display resolutions.\n\nIf unsure, select Off.");
   static const char TR_3D_DEPTH_DESCRIPTION[] =
-      QT_TR_NOOP("Controls the separation distance between the virtual cameras.\nA higher value "
+      QT_TR_NOOP("Controls the separation distance between the virtual cameras.\n\nA higher value "
```

Comment:
Could anyone with more knowledge about this feature tell what a recommended setting would be?

Reply:
It's dependent on screen size, distance from screen, IPD, etc. Basically there is no recommended setting and the user must tweak it to their liking.

Reply:
What about a message along the lines of "Adjust to whichever value provides the best 3D viewing experience."?

Reply:
I don't think any additional information is needed, as the rest of the text explains it pretty concisely.